src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -816,185 +825,198 @@ public void postBlUsersLogin(boolean expandPerms, List<String> include, String u
           .ifPresent(header -> headers.put(HttpHeaders.USER_AGENT, header));
         Optional.ofNullable(xForwardedFor)
           .ifPresent(header -> headers.put(X_FORWARDED_FOR_HEADER, header));
-        loginResponse[0] = client.request(HttpMethod.POST, entity, moduleURL, headers);
```

Comment:
Why change the way the  login client is constructed and used for this PR? Nothing appears to have changed with the parameters.